src/app.js

```diff
@@ -21,6 +21,7 @@ const optionalProperties = [
     'platform.counts.projectSnapshots',
     'platform.counts.projectTemplates',
     'platform.counts.projectStacks',
+    'platform.counts.blueprints',
```

Comment:
You'll need a migration too I assume, and a column that defaults to zero?

Reply:
Yeah - released just after pushing. Will add shortly